scipy/signal/signaltools.py

```diff
@@ -1917,11 +1916,19 @@ def resample_poly(x, up, down, axis=0, window=('kaiser', 5.0)):
     the number of operations during polyphase filtering will depend on
     the filter length and `down` (see `scipy.signal.upfirdn` for details).
 
-    The `window` argument is passed directly to `scipy.signal.firwin`
-    to design a low-pass filter.
+    The argument `window` specifies the FIR low-pass filter design.
+
+    If `window` is an array_like it is assumed to be the FIR filter
+    coefficients. Note that the FIR filter is applied after the upsampling
+    step, so it should be designed to operate on a signal at a sampling
+    frequency higher than the original by a factor of `up//gcd(up, down)`.
```

Comment:
We should probably also mention that the output will be centered with respect to this array, so it might be best to pass a symmetric filter with an odd number of samples to ensure a zero-phase filter.